src/platform/common/utils/localize.ts

```diff
@@ -808,7 +808,7 @@ export namespace DataScience {
                 key: 'DataScience.pandasRequiredForViewing',
                 comment: ["{Locked='pandas'", 'This is the name of the pandas package']
             },
-            "Python package 'pandas' is required for viewing data."
+            "Python package 'pandas' is required for viewing data. Please ensure you have installed 'pandas' version {0} or above."
```

Comment:
You need to update the package.nls.json file as well, coudlnt see that change in this PR